vmdb/lib/miq_automation_engine/models/miq_ae_class_copy.rb

```diff
@@ -78,15 +78,14 @@ def add_fields
 
   def validate
     dest_class = MiqAeClass.find_by_fqname("#{@target_ns_fqname}/#{@target_name}")
-    if dest_class
-      dest_class.destroy if @overwrite
-      raise "Destination Class already exists #{dest_class.fqname}" unless @overwrite
-    end
+    return unless dest_class
+    dest_class.destroy if @overwrite
```

Comment:
@kbrock I had coded this with an if/else but rubocop complains about not using a 'guard clause' so I ended up changing it to a singe if with a raise